webapp/src/components/llmbot_post.tsx

```diff
@@ -10,37 +10,15 @@ import {SendIcon} from '@mattermost/compass-icons/components';
 
 import {doPostbackSummary, doRegenerate, doStopGenerating} from '@/client';
 
-import {useSelectNotAIPost, useSelectPost} from '@/hooks';
+import {useSelectNotAIPost} from '@/hooks';
 
 import {PostMessagePreview} from '@/mm_webapp';
 
 import PostText from './post_text';
 import IconRegenerate from './assets/icon_regenerate';
 import IconCancel from './assets/icon_cancel';
 
-const FixPostHover = createGlobalStyle<{disableHover?: string}>`
-	${(props) => props.disableHover && css`
-	&&&& {
-		[data-testid="post-menu-${props.disableHover}"] {
-			display: none !important;
-		}
-		[data-testid="post-menu-${props.disableHover}"]:hover {
-			display: none !important;
-		}
-	}`}
-`;
-
-const PostBody = styled.div<{disableHover?: boolean}>`
-	${(props) => props.disableHover && css`
-	::before {
-		content: '';
-		position: absolute;
-		width: 110%;
-		height: 110%;
-		left: 50%;
-		top: 50%;
-		transform: translate(-50%, -50%);
-	}`}
+const PostBody = styled.div`
```

Comment:
0/5 here, I see the value on giving a name to the div itself, but maybe, because it is no longer needed from the styles perspective we can remove this. Anyway, I'm ok leaving it as is, just to give it a name.

Reply:
The name is helpful for debugging and whatnot I think we can leave it.